proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/sdk/PowerAuthSDK.java

```diff
@@ -1670,9 +1654,11 @@ public void authenticateUsingFingerprint(Context context, FragmentManager fragme
      * @param description Dialog description.
      * @param forceGenerateNewKey Pass true to indicate that a new key should be generated in Keystore
      * @param callback Callback with the authentication result.
+     * @return {@link ICancelable} object associated with the biometric prompt.
      */
     @RequiresApi(api = Build.VERSION_CODES.M)
-    private void authenticateUsingFingerprint(final @NonNull Context context, final @NonNull FragmentManager fragmentManager, final @NonNull String title, final @NonNull String description, final boolean forceGenerateNewKey, final IFingerprintActionHandler callback) {
+    @NonNull
+    private ICancelable authenticateUsingFingerprint(final @NonNull Context context, final @NonNull FragmentManager fragmentManager, final @NonNull String title, final @NonNull String description, final boolean forceGenerateNewKey, final IBiometricAuthenticationCallback callback) {
```

Comment:
`authenticateUsingBiometry`?

Reply:
The same as above with public version of the method. I've changed that to `authenticateUsingBiometry` together with documentation update.